### ISSUE TITLE
cannot inherit from non-open class RegexRule outside of its defining module

### ISSUE THREAD

**Issue by kiwo12345** [jpotts18/SwiftValidator#162]
```
import SwiftValidator
class NumberRule: RegexRule {

    //Only allow integers
    static let regex = "^[0-9]*$"

    convenience init(message : String = "Not a valid number"){
        self.init(regex: NumberRule.regex, message : message)
    }
}
```

Must RegexRule be open instead of public? : 

> Swift 3 is adding 2 more access levels (open and fileprivate) and changing the meaning of private:
> 
> private: symbol visible within the current declaration only.
> fileprivate: symbol visible within the current file.
> internal: symbol visible within the current module.
> public: symbol visible outside the current module.
> open: for class or function to be subclassed or overridden outside the current module.

**Comment by kompw** [comment]
+1)

**Comment by happiehappie** [comment]
+1

**Comment by cikpis** [comment]
+1

I need to extend Validator class.

**Comment by srmds** [comment]
👍 👍 👍 👍 👍 👍

**Comment by davepatterson** [comment]
This was fixed in #158
